adcm_aio_client/core/objects/cm.py

```diff
@@ -81,6 +86,32 @@ def get_own_path(self: Self) -> Endpoint:
 class BundlesNode(PaginatedAccessor[Bundle, None]):
     class_type = Bundle
 
+    async def _download_external_bundle(self: Self, url: str) -> bytes:
+        try:
+            urlopen(url)  # noqa S310
+            async with httpx.AsyncClient() as client:
+                response = await client.get(url)
+                return response.content
+        except ValueError as err:
+            raise OperationError from err
```

Comment:
cryptic

please explain

Reply:
urlopen was meant to checj whether the string is real url